server/src/project/project.service.ts

```diff
@@ -283,8 +284,50 @@ export class ProjectService {
   constructor(
     private readonly dynamicsWebApi: OdataService,
     private readonly config: ConfigService,
+    private readonly carto: CartoService,
   ) {}
 
+  async getBblsGeoJSON(bbls) {
+    let collectedBBLs = bbls;
+
+    if (collectedBBLs === null) return null;
+
+    collectedBBLs = collectedBBLs.filter(bbl => bbl !== null);
```

Comment:
I'd Make the arg in teh func signature `bbls = []` with a default.

Reply:
why are there null BBLs at all?

Reply:
Good idea on the default args. Hmm yeah good q... That was a copy from old code, maybe the original author noticed sometimes null was included in the bbls array? not sure. I haven't seen it myself. We could remove it for now and watch out for edge cases